jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -834,28 +832,38 @@ function Test-JumpCloudUsername {
             # Write-ToLog "Searching JC for: $Username"
             $Response = Invoke-WebRequest -Method 'Post' -Uri "https://console.jumpcloud.com/api/search/systemusers" -Headers $Headers -Body $Body -UseBasicParsing
             $Results = $Response.Content | ConvertFrom-Json
+
             $StatusCode = $Response.StatusCode
         } catch {
             $StatusCode = $_.Exception.Response.StatusCode.value__
+            Write-ToLog -Message "Status Code $($StatusCode)"
         }
     }
     End {
         # Search User should return 200 success
         If ($StatusCode -ne 200) {
-            Return $false, $null
             Write-ToLog -Message "JumpCloud username could not be found"
+            Return $false, $null, $null, $null
         }
         If ($Results.totalCount -eq 1 -and $($Results.results[0].username) -eq $Username) {
             # write-host $Results.results[0]._id
             Write-ToLog -Message "Identified JumpCloud User`nUsername: $($Results.results[0].username)`nID: $($Results.results[0]._id)"
-            return $true, $Results.results[0]._id
+            if ($Results.results[0].SystemUsername) {
```

Comment:
the verbose output here is a little messy for me. It's printing the results twice if a system username is found. 

![Screen Shot 2023-05-24 at 1 18 00 PM](https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/54448601/dee4b07a-dc19-45ac-91e6-ffaf4bc9dca6)

I'd suggest just printing "The JumpCloud User: $(username) has a local user account: $(systemUsername), the $(SelectedUserName) will be migrated and converted to $(Env:COMPUTERNAME\$(systemUsername))